### DISCUSSION TITLE
Need more information about nchain

### DISCUSSION THREAD

**Discussion by raingggg** [guydavis/machinaris#373] [Q&A] [answered]
OS: ubuntu 20.0
Machinaris: 0.6.1
Plots: chia NFT plots got 0 reward

I have waited 4 times longer than the expected win time, and got 0 reward now. This is a bit of confusing. Then I try to understand more about the nchain. However on the website, it says quite few about how would the nchain blockchain work. It would be good if there is any more information about it.

At least I would like to know:
1. Whether there is a whitebook for nchain or not - cannot find it from the website https://ext9.org/
2. How to get latest news from nchain team? - cannot join discord from the above site discord link
3. Total supply of the coin?
4. The inflation mechnism?
5. Any differences with chia chain?

**Comment by guydavis** [comment, marked as answer]
Hi, please see [Chia Forks](https://xchforks.com/n-chain).  Lots of good info there including Discord link.  

Each fork supported by Machinaris is completely optional, so only launch the fork containers you want to run.

Please mark this Discussion as Answered.  Thanks!
